src/speeds.jl

```diff
@@ -1,3 +1,27 @@
+export Alfven_velocity, Alfven_speed
+
+function Alfven_velocity(B::BField, ρ::Density)
+    return B / sqrt(μ0 * ρ) |> upreferred
+end
+
+function Alfven_velocity(B::BField, n::NumberDensity; mass_numb = 1, Z = 1)
+    ρ = n * (mass_numb * mp + Z * me)
+    return Alfven_velocity(B, ρ)
+end
+
+Alfven_velocity(B::AbstractVector, args...; kwargs...) =
+    [Alfven_velocity(Bi, args...; kwargs...) for Bi in B]
+
+"""
+    Alfven_speed(args...; kwargs...)
```

Comment:
Let's make this type signature more informative by specifying what args and kwargs are.